spec/integration/aws_route53_hosted_zone_spec.rb

```diff
@@ -64,6 +64,17 @@
             }.to create_an_aws_route53_hosted_zone(zone_name,
                                                    config: { comment: expected_comment }).and be_idempotent
           end
+
+          it "updates the zone comment when none is given" do
+            expect_recipe {
+              aws_route53_hosted_zone zone_name do
+                comment "Initial comment."
+              end
+              aws_route53_hosted_zone zone_name do
+              end
+            }.to create_an_aws_route53_hosted_zone(zone_name,
+                                                   config: { comment: nil }).and be_idempotent
```

Comment:
Here's a snippit:

```
[2015-11-11T01:21:26+00:00] INFO: [Aws::Route53::Client 400 0.028024 0 retries] update_hosted_zone_comment(id:"MY-ZONE-ID",comment:nil) Aws::Route53::Errors::MalformedInput Unexpected complex element termination



    ================================================================================
    Error executing action `create` on resource 'aws_route53_hosted_zone[EXAMPLE.COM] (/hostedzone/MY-ZONE-ID)'
    ================================================================================

    Aws::Route53::Errors::MalformedInput
    ------------------------------------
    Unexpected complex element termination
```

By not passing a `comment`, the default is used.  If you want to be explicit then it can be written as:

```
aws_route53_hosted_zone zone_name do
  comment nil
end
```
The awkward thing is that `nil` still comes back from the API.